src/app/layout/header/header.component.html

```diff
@@ -3,14 +3,21 @@
 
   @if ((authStore.selectIsAuthenticated$ | async) === true) {
     <ul class="nav-bar">
+      @for (item of menuAuth; track $index) {
+        <li class="nav-item__active">
```

Comment:
`.nav-item__active` only use when this menu item is activated. Research `routerLinkActive` and apply